go/engine/identify2_with_uid.go

```diff
@@ -343,23 +343,29 @@ func (e *Identify2WithUID) runIdentifyUI(ctx *Context) (err error) {
 	// fingerprints and the user's UID and username.
 	e.remotesReceived = e.them.BaseProofSet()
 
+	e.G().Log.Debug("| IdentifyUI.Start(%s)", e.them.GetName())
```

Comment:
yeah, well pretty well-tested and lots of logging around it....  just went with this for now...